crowbar_framework/app/controllers/deploy_queue_controller.rb

```diff
@@ -25,6 +25,6 @@ def currently_deployed
   end
 
   def deployment_queue
-    Chef::DataBag.load("crowbar/queue")["proposal_queue"] rescue []
+    ProposalQueue.find("queue").proposals rescue []
```

Comment:
I prefer to handle this case in the proposals method, otherwise we tend to a fail driven code.